### ISSUE TITLE
Config: `ignoreSendTextErrors`

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1875]
Some people do not expect their whole node process to crash when a sendtext error is thrown.

Ideally, all client methods are surrounded with `try/catch` but there should be an option to downgrade these sendText errors to just warnings

**Comment by smashah** [comment]
`config.ignoreSendTextErrors` was not implemented. Behaviour is now inferred from `onError` where the default is `NOTHING` (i.e don't throw the error; do nothing).

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci minor`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.9.0](https://github.com/open-wa/wa-automate-nodejs/compare/4.8.2...4.9.0) (2021-07-13)

- build(deps-dev): bump @types/shelljs from 0.8.8 to 0.8.9 [`#1852`](https://github.com/open-wa/wa-automate-nodejs/pull/1852)
- build(deps-dev): bump @types/marked from 2.0.3 to 2.0.4 [`#1853`](https://github.com/open-wa/wa-automate-nodejs/pull/1853)
- build(deps-dev): bump ts-node from 10.0.0 to 10.1.0 [`#1884`](https://github.com/open-wa/wa-automate-nodejs/pull/1884)
- build(deps-dev): bump @typescript-eslint/eslint-plugin from 4.28.2 to 4.28.3 [`#1886`](https://github.com/open-wa/wa-automate-nodejs/pull/1886)
- build(deps): bump socket.io from 4.1.2 to 4.1.3 [`#1887`](https://github.com/open-wa/wa-automate-nodejs/pull/1887)
- build(deps-dev): bump @types/command-line-usage from 5.0.1 to 5.0.2 [`#1849`](https://github.com/open-wa/wa-automate-nodejs/pull/1849)
- build(deps): bump ts-morph from 11.0.0 to 11.0.3 [`#1888`](https://github.com/open-wa/wa-automate-nodejs/pull/1888)
- build(deps): bump socket.io-client from 4.1.2 to 4.1.3 [`#1885`](https://github.com/open-wa/wa-automate-nodejs/pull/1885)
- build(deps): bump @discordjs/collection from 0.1.6 to 0.2.0 [`#1882`](https://github.com/open-wa/wa-automate-nodejs/pull/1882)
- build(deps-dev): bump @types/command-line-args from 5.0.0 to 5.0.1 [`#1854`](https://github.com/open-wa/wa-automate-nodejs/pull/1854)
- build(deps-dev): bump @types/is-url from 1.2.29 to 1.2.30 [`#1883`](https://github.com/open-wa/wa-automate-nodejs/pull/1883)
- build(deps-dev): bump @types/fs-extra from 9.0.11 to 9.0.12 [`#1851`](https://github.com/open-wa/wa-automate-nodejs/pull/1851)
- build(deps): bump pino from 6.11.3 to 6.12.0 [`#1881`](https://github.com/open-wa/wa-automate-nodejs/pull/1881)
- build(deps-dev): bump typedoc from 0.21.2 to 0.21.4 [`#1880`](https://github.com/open-wa/wa-automate-nodejs/pull/1880)
- build(deps): bump type-fest from 1.2.1 to 1.2.2 [`#1879`](https://github.com/open-wa/wa-automate-nodejs/pull/1879)
- build(deps-dev): bump command-line-args from 5.1.1 to 5.1.3 [`#1878`](https://github.com/open-wa/wa-automate-nodejs/pull/1878)
- build(deps-dev): bump @typescript-eslint/parser from 4.28.2 to 4.28.3 [`#1877`](https://github.com/open-wa/wa-automate-nodejs/pull/1877)
- build(deps-dev): bump @types/pino from 6.3.8 to 6.3.9 [`#1855`](https://github.com/open-wa/wa-automate-nodejs/pull/1855)
- build(deps-dev): bump @types/line-reader from 0.0.33 to 0.0.34 [`#1876`](https://github.com/open-wa/wa-automate-nodejs/pull/1876)
- build(deps-dev): bump @types/puppeteer from 5.4.3 to 5.4.4 [`#1856`](https://github.com/open-wa/wa-automate-nodejs/pull/1856)
- build(deps): bump ts-json-schema-generator from 0.94.0 to 0.94.1 [`#1857`](https://github.com/open-wa/wa-automate-nodejs/pull/1857)
- build(deps-dev): bump @types/death from 1.1.1 to 1.1.2 [`#1862`](https://github.com/open-wa/wa-automate-nodejs/pull/1862)
- build(deps-dev): bump @types/node from 16.0.0 to 16.3.1 [`#1870`](https://github.com/open-wa/wa-automate-nodejs/pull/1870)
- Release 4.9.0 [`753b527`](https://github.com/open-wa/wa-automate-nodejs/commit/753b527adb763c9e8ecaa3b9d1d13bfd06209053)
- 📦  lock update [`f13918e`](https://github.com/open-wa/wa-automate-nodejs/commit/f13918ec5f1aa6464d8a3347f29e25072dd495a4)
- ✨ feat: `clearAllChats` now has ts parameter #1869 [`e839fbe`](https://github.com/open-wa/wa-automate-nodejs/commit/e839fbeb99b595dbda0219ca60225b113fa5846d)
- 🏷️  fix some collector typings [`b2f9a2b`](https://github.com/open-wa/wa-automate-nodejs/commit/b2f9a2b946a8eb1e2d55c99e0ed3014831b0d741)
- 🥚 patches update [`398fa31`](https://github.com/open-wa/wa-automate-nodejs/commit/398fa31c699bba485d90bbdd67808308afebb430)
- 🥅  Do not throw an error on `sendText` unless `config.onError` == `"THROW"` #1875 [`40d4b55`](https://github.com/open-wa/wa-automate-nodejs/commit/40d4b554f086bef4e9620d334b87c63e52c1b6b2)
- 🎁 GA: `clearAllChats` + tagging participants in media #1865 [`701c0a2`](https://github.com/open-wa/wa-automate-nodejs/commit/701c0a29dff4e48bde82404dbd5590a98655bac7)
- 🥚  patches update [`e2d91be`](https://github.com/open-wa/wa-automate-nodejs/commit/e2d91be19dac0bccd622d3bc6db31b40b3c73a42)
- 🥚  patches update [`05e87b5`](https://github.com/open-wa/wa-automate-nodejs/commit/05e87b55877578a32fa69b8c78410f39dc08757c)
- 🔊  state reason why `client.kill()` is being called #1873 [`c09e1db`](https://github.com/open-wa/wa-automate-nodejs/commit/c09e1dbbf6329b0afe7079e3cb3d5c3fc73714ea)
- 🥚 patches update [`2b31831`](https://github.com/open-wa/wa-automate-nodejs/commit/2b31831adfb9430a9913c9b06737e12e5fd7ac58)
- 🥚 patches update [`062395f`](https://github.com/open-wa/wa-automate-nodejs/commit/062395f95ee75cd20491b03c7f293f5bf299190f)
- 🥚 patches update [`0c41030`](https://github.com/open-wa/wa-automate-nodejs/commit/0c410306617e613264c8b3d0e9ec7d5fa3be39b5)
- 🥚 patches update [`3a51af2`](https://github.com/open-wa/wa-automate-nodejs/commit/3a51af235dfb2a99fba696c49d54a65daec6f754)
